tools/travis/linux_release/after_success.sh

```diff
@@ -1,21 +1,32 @@
 #!/bin/bash
 #set -ev
-
 if [[ "${TRAVIS_PULL_REQUEST}" == "false" ]]; then
     echo -e "Packaging binaries and libs"
 
-    # === Copy Libs ===
-	QT_LIB_DIR=/opt/qt510/lib
-    QT_LIBS=(libQt5Charts libQt5Concurrent libQt5Core libQt5Gui libQt5Network libQt5PrintSupport libQt5Qml libQt5Svg libQt5Test libQt5Widgets libQt5Xml libQt53DCore libQt53DExtras libQt53DInput libQt53DLogic libQt53DRender )
-    n_elements=${#QT_LIBS[@]}
-    for ((i = 0; i < n_elements; i++)); do
-        libpath="$QT_LIB_DIR/${QT_LIBS[i]}.*"
-        cp $libpath ./lib
-    done
+    # === Linux Deploy Qt ===
+	#sourcing qt510 again, this is for linuxdeployqt; verified redundant, so commented out. 
```

Comment:
Please mind indentation

Reply:
Corrected spacing, and used spaces instead of tabs.